test/files/run/t5717.scala

```diff
@@ -20,7 +20,8 @@ object Test extends StoreReporterDirectTest {
     val List(i) = filteredInfos
     // for some reason, nio doesn't throw the same exception on windows and linux/mac
     val path = if(util.Properties.isWin)"\\a" else "/a"
-    val expected = s"error writing ${testOutput.path}/a/B.class: Can't create directory ${testOutput.path}${path}" +
+    import File.pathSeparator
+    val expected = s"error writing ${testOutput.path}${pathSeparator}a${pathSeparator}B.class: Can't create directory ${testOutput.path}${pathSeparator}/a" +
```

Comment:
There is a residual forward slash.

The other time-tested method is just to replaceAll backslashes with forward slash to normalize the text.

Reply:
That was me recommending test string techniques to retronym.